ts/src/profiler.ts

```diff
@@ -228,14 +228,13 @@ export class Profiler extends common.ServiceObject {
     try {
       prof = await this.createProfile();
     } catch (err) {
-      this.logger.error(
-          `Error requesting profile type to be collected: ${err}.`);
+      this.logger.error(`Failed to create profile: ${err}.`);
       return this.config.backoffMillis;
     }
     try {
       await this.profileAndUpload(prof);
     } catch (err) {
-      this.logger.error(`Error collecting and uploading profile: ${err}.`);
+      this.logger.error(`Failed to collect and upload profile: ${err}.`);
```

Comment:
@aalexand -- should I do a full-stop here?
Some errors don't end with a full stop and some do. Should I add a full stop only if the error doesn't have one? Or put the error in quotes?

Reply:
I would drop the full stop here.

Also, if the error message starts with a capital letter, the text will read weird here with the capital letter after the colon: `Failed to do foo: Failed to connect to server bar.`. This is why Go has the convention that the error messages start with a lowercase, so that the chaining them in the log messages looks good. I don't know whether there is a Node.js style for this, but if there is we should follow it.

Reply:
I think I started with lower-case error messages, and changed because Node.js convention is to capitalize errors. Errors I've seen generated by other code are capitalized.